backend/app/Services/WorkOrder/WorkOrderService.php

```
@@ -56,6 +63,35 @@ public function createWorkOrder(array $data): WorkOrder
+    /**
+     * Merge an immutable `revision` block into the process snapshot for the given
+     * product revision (#180). Returns the snapshot unchanged when no revision is
+     * selected; initialises an array snapshot when the order has no BOM/template.
+     */
+    private function attachRevisionSnapshot(?array $snapshot, ?int $revisionId): ?array
+    {
+        if (! $revisionId) {
+            return $snapshot;
+        }
+
+        $revision = \App\Models\ProductRevision::find($revisionId);
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\ProductRevision::find().


(staticMethod.notFound)